src/tts_text/phoneme_covering.py

```diff
@@ -0,0 +1,194 @@
+"""Module for creating a phoneme covering set."""
+
+import json
+import re
+import logging
+from collections import Counter
+from pathlib import Path
+from tqdm.auto import tqdm
+
+from datasets import load_dataset, Dataset
+from omegaconf import DictConfig
+
+logger = logging.getLogger(__name__)
+
+
+def sort_wiki_by_phoneme_occurence(cfg: DictConfig) -> Dataset:
```

Comment:
The function name is misleading - it sounds like you give it a wiki dataset and it sorts it, but it actually loads it from scratch. Maybe something like `load_sorted_wikipedia_dataset` instead?

Reply:
Docstring should reflect this change.